src/main/java/org/reaktivity/nukleus/http/internal/routable/stream/ServerAcceptReplyState.java

```diff
@@ -19,36 +19,34 @@
 
 import org.reaktivity.nukleus.http.internal.routable.Target;
 
-final class ServerConnectionState
+final class ServerAcceptReplyState
 {
     final long streamId;
-    String target;
+    final Target loopBackTarget;
```

Comment:
I called it loopBackTarget to make it clear it's the target for short circuiting responses back to the source, distinct from the reply target that will be used when a route has been found and applied. Of course once we do the many to one changes these will be one and the same.